src/common/drpc.c

```diff
@@ -562,24 +603,27 @@ send_response(struct drpc *ctx, Drpc__Response *response)
 static int
 get_incoming_call(struct drpc *ctx, Drpc__Call **call)
 {
-	int		rc;
-	uint8_t		*buffer;
-	size_t		buffer_size = UNIXCOMM_MAXMSGSIZE;
-	ssize_t		message_len = 0;
+	struct drpc_alloc	alloc = PROTO_ALLOCATOR_INIT(alloc);
+	int			rc;
+	uint8_t			*buffer;
+	size_t			buffer_size = UNIXCOMM_MAXMSGSIZE;
+	ssize_t			message_len = 0;
 
 	D_ALLOC(buffer, buffer_size);
 	if (buffer == NULL)
 		return -DER_NOMEM;
 
 	rc = unixcomm_recv(ctx->comm, buffer, buffer_size,
 				&message_len);
-	if (rc != DER_SUCCESS) {
+	if (rc != 0) {
```

Comment:
in other cases , you are adding comparisons with  -DER_SUCCESS, why replace with 0 here and elsewhere in this file?

Reply:
i.e. https://github.com/daos-stack/daos/pull/2977/files#diff-53e5a32e6082d08bd2765e6f1276c678R349

Reply:
I normally compare against -DER_SUCCESS, however @kjacque pointed out that in this file all the comparisons are against 0, so I changed the ones I added in drpc.c to be like that, but elsewhere I think comparison again DER_SUCCESS is more common.